apps/core-app/src/components/customFields/tributeInput.tsx

```diff
@@ -296,11 +272,13 @@ const TemporaryWarning = ({
   };
 
   return (
-    <TempWarningBox>
-      <ParXs>You must approve {tokenName || 'Token'} to submit</ParXs>
-      <WarningButton sm type="button" onClick={handleApprove}>
+    <FieldAlert
+      className="warning"
+      message={`You must approve ${tokenName || 'Token'} to submit`}
+    >
+      <Button sm type="button" onClick={handleApprove}>
         {txState === TxStates.Loading ? 'Loading...' : 'Approve'}
-      </WarningButton>
-    </TempWarningBox>
+      </Button>
+    </FieldAlert>
```

Comment:
one thing im curious about here is if this is how we would want a `FeildAlert` to act - basically if you pass it a `ReactNode`, that ReactNode will be presented on the Right side of the component.

Reply:
Interesting. In these situations, I generally use prop names that describe where things are going to go  (ex. renderLeft or renderRight) but this works too.